corehq/apps/accounting/tasks.py

```diff
@@ -368,7 +368,7 @@ def create_wire_credits_invoice(domain_name,
         record.save()
 
 
-@task(ignore_result=True)
+@task(ignore_result=True, acks_late=True)
```

Comment:
looks like the email gets sent as the last thing the function does so guessing this is pretty safe